gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx

```diff
@@ -20,6 +22,21 @@ export const MapMenu = (props: MapMenuProps) => {
   const {markers, menuItem} = useContext(MapContext);
   const markerTypes = Array.from(new Set(markers.map(marker => marker.markerType)));
 
+  const [currentIdx, setCurrentIdx] = useState(-1);
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
It is not clear from the name what is being opened

Reply:
Change to isMenuItemOpen